frontend/src/metabase/nav/components/SearchBar.tsx

```diff
@@ -16,7 +16,7 @@ import { zoomInRow } from "metabase/query_builder/actions";
 
 import { getSetting } from "metabase/selectors/settings";
 import RecentsList from "metabase/nav/components/RecentsList";
-import { SearchFilterModal } from "metabase/search/components/SearchFilterModal/SearchFilterModal";
+import { SearchFilterModal } from "metabase/search/components/SearchFilterModal/SearchFilterModal/SearchFilterModal";
```

Comment:
nit: I think you should do something about this `SearchFilterModal` repeating 3 times in the path. It's starting to get confusing.